README.md

```diff
@@ -49,6 +49,17 @@ sed -i "s/<REPLACE ME>/GITHUB_NICK/g" boxes.d/99-local.yaml
 vagrant up centos7-devel
 ```
 
+In case you don't want to use vagrant, you can still use ansible playbooks and roles from this repo. This is useful if you have existing host, e.g. a beaker machine. Let's say you have host test.example.com on which you want to deploy devel environment, following should do the trick:
+
+```sh
+echo "[devel]\ntest.example.com" > local_inventory
+ansible-playbook --private-key=/home/ares/.ssh/id_rsa --user root --inventory local_inventory --extra-vars katello_devel_github_username=katello playbooks/devel.yml
+```
```

Comment:
Just a thought, if we recommend using local_inventory, should we also add it to .gitignore to prevent people from accidentally pushing it?

Reply:
added, though I guess people will use random names for these files

Reply:
This will require user to specify `-i local_inventory` in each command  given (https://github.com/theforeman/forklift/blob/master/ansible.cfg#L6). I've been thinking about this prior to this and thought moving to an inventory folder at the top level would help. This would allow us to create some other inventories potentially as well as support a user structure for inventories.

Reply:
do you want to have inventories dir ignored in git or have separate user_inventories as we have it for playbooks?

Reply:
As far as I know, you can only specify a single inventory dir so it'd be specifically named inventory(ies) that are .gitignored contained within that folder. That would simplify things for users.

Reply:
I hope I got it right that we should have `inventories` dir which is listed in .gitignore (though it should exists in repo for initial clone I guess)